src/svd/_device.py

```diff
@@ -194,7 +201,10 @@ def __reversed__(self) -> Iterator[T]:
             yield self._getter(k)
 
 
-class LazyFixedMapping(Mapping[str, T]):
+VT = TypeVar("VT")
```

Comment:
nit again: A type that seems very arbitrary without a note or comment as to what it means.